libsupport/include/katana/Strings.h

```diff
@@ -33,6 +34,13 @@ KATANA_EXPORT std::string TrimSuffix(
 
 KATANA_EXPORT bool HasSuffix(const std::string& s, const std::string& suffix);
 
+/// SplitView returns a list of words in \param s using \param sep as the
+/// delimiter string. Splits at most \param max times (so there will be at most
+/// max + 1 entries in the output)
+KATANA_EXPORT std::vector<std::string_view> SplitView(
```

Comment:
Nice comment.

Reply:
I don't think doxygen `param` supports inline formatting

https://www.doxygen.nl/manual/commands.html#cmdparam

so I think this will render as


```
SplitView returns a list of words in.

s | using
sep | as the delimiter string. Splits at most
max | times (so there will be at most max + 1 entries in the output)
```

Reply:
Oh. Woops! i've been using this style for months. Should have looked at the docs.